AndroidTest/app/build.gradle

```diff
@@ -64,14 +64,14 @@ dependencies {
     // since our test harness is a standard android app which will run our tests,
     // we include the dependencies necessary for build and test here
 
-    compile group: 'com.fasterxml.jackson.core', name: 'jackson-databind', version: '2.1.1'
-    compile group: 'com.google.guava', name: 'guava', version: '15.0'
-    compile group: 'commons-io', name: 'commons-io', version: '2.4'
-    compile group: 'commons-codec', name: 'commons-codec', version: '1.9'
-    compile group: 'org.apache.commons', name: 'commons-lang3', version: '3.3.2'
+    compile 'com.fasterxml.jackson.core:jackson-databind:2.1.1'
+    compile 'com.google.guava:guava:15.0'
+    compile 'commons-io:commons-io:2.4'
+    compile 'commons-codec:commons-codec:1.9'
+    compile 'org.apache.commons:commons-lang3:3.3.2'
     compile 'com.google.code.findbugs:jsr305:3.0.0' //this is needed for some versions of android
     compile files('../../cloudant-sync-datastore-android-encryption/libs/sqlcipher.jar') //required sqlcipher lib
-
+    compile files('../../cloudant-sync-datastore-android/libs/android-support-v4.jar')
```

Comment:
Yes, it can be used from a local repository, but I think as you say the issue is that building outside Android Studio then becomes an issue. Unfortunately it doesn't seem to be uploaded to any official public repository.